cmd/reconciler-manager/main.go

```diff
@@ -107,6 +107,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	otelSA := controllers.NewOtelSAReconciler(*clusterName, mgr.GetClient(),
+		ctrl.Log.WithName("controllers").WithName("OtelSA"),
```

Comment:
Extract this into a var?

Reply:
What is your concern here?

Reply:
Are you suggesting making a var for `OtelSA`?